README.md

```diff
@@ -1,87 +1,94 @@
+# Table of Contents
+* [What is Archipelago?](https://github.com/esmero/archipelago-documentation#archipelago-commons-intro)
+* [Code of Conduct](https://github.com/esmero/archipelago-documentation#code-of-conduct)
```

Comment:
I think it would be better, Instead of linking [Code of Conduct] to an anchor down, make this a direct link to the CODE_OF_CONDUCT.md / That way code of conduct does not end mixed up with the guides. That way all after line 64 does not need to be shifted down neither. What do you think about that?